doc/changelog/08-vernac-commands-and-options/16126-extraction-output.rst

```diff
@@ -0,0 +1,7 @@
+- **Added:**
+  :opt:`Extraction Output Directory` option for specifying the
+  directory in which `.ml` and `.mli` files will be appear during
+  extraction
```

Comment:
```suggestion
  directory where `.ml` and `.mli` files will be written for extractions
```

Reply:
Since we talk about the process "of extraction" I didn't want to change the wording at all. Keeping it nouny rather than verby.

Reply:
"files will be appear during extraction" in ungrammatical.  Please fix.